Review section for ComboBox control. (#610)
telerik/maui-docs: controls/combobox/templates.md

# .NET MAUI ComboBox Templates

If the default templates of the control do not suit your needs, you can define custom ones. The available templates for customizing are:

* `ItemTemplate`(`DataTemplate`)&mdash;Defines the template of the items that are visualized in the dropdown list.

> When the selection mode is single and the control is not editable if there is ItemTemplate set the same template will be visualized in the box part of the control when item is selected.

* `SelectedItemTemplate`(`DataTemplate`)&mdash;Defines the template of the selected items that are visualized in the dropdown list.
* `TokenTemplate`(`DataTemplate`)&mdash;Defines the template of the tokens that are visualized when multiple selection is performed.

> * `ShowMoreTemplate`(`DataTemplate`)&mdash;Defines the Template of the show more UI that gets visualized when the control is not focused and there is not space for all Tokens to be visualized when the selection mode is multiple.

Reviewer: ```suggestion
* `ShowMoreTemplate`(`DataTemplate`)&mdash;Defines the Template of the Show More UI that gets visualized when the control is not focused and the space is not enough to show all tokens when using the Multiple selection mode.
```

> when the default TokenTemplate is overridden, you will need to implement custom logic for removing the tokens from the ComboBox:

> here is a sample logic removing the token when adding `TapGestureRecognizer` to the Label:

Reviewer: capital letter
